bibtex-actions.el

```diff
@@ -341,8 +370,14 @@ is non-nil, also run the `bibtex-actions-before-refresh-hook' hook."
   (interactive "P")
   (when force-rebuild-cache
     (run-hooks 'bibtex-actions-force-refresh-hook))
+  ;; t in the result indicates that cache was computed but came out nil nil
+  ;; itself is used to indicate a cache that hasn't been computed this is to
+  ;; work around the fact that we can't differentiate between empty list and nil
   (setq bibtex-actions--candidates-cache
-        (bibtex-actions--format-candidates)))
+        ((lambda (x) (if x x t)) (bibtex-actions--format-candidates)))
+  (let (bibtex-completion-bibliography (bibtex-actions--local-files-to-cache))
+    (setq bibtex-actions--local-candidates-cache
```

Comment:
This is wrong, as that's not a variable.

Reply:
Yes, I missed a pair of parenthesis here